src/actions/ArenaMatchEnd.ts

```diff
@@ -2,7 +2,7 @@ import { ILogLine } from "../types";
 
 export interface ArenaMatchEndInfo {
   timestamp: number;
-  winningTeamId: string;
+  winningTeamId: number;
```

Comment:
If we change the type here we will likely need to migrate Firestore data over; our gql interface will have to become `winningTeamId: String! | number!` which will mean the generated type on the client will become `string | number`; a difficult type to use in components imo